common/presets.ts

```diff
@@ -199,6 +200,20 @@ export const serviceGenMap: Record<Exclude<ChatAdapter, 'default'>, GenMap> = {
     gaslight: 'gaslight',
     oaiModel: 'oaiModel',
   },
+  claude: {
+    maxTokens: 'max_tokens_to_sample',
+    repetitionPenalty: '',
+    repetitionPenaltyRange: '',
+    repetitionPenaltySlope: '',
+    tailFreeSampling: '',
+    temp: 'temperature',
+    topK: '',
+    topP: '',
+    typicalP: '',
+    topA: '',
+    gaslight: 'gaslight',
+    claudeModel: 'claudeModel'
+  },
```

Comment:
Can you teach me what is this part used for?

Reply:
@sceuick can probably provide a more holistic explanation but my understanding is that this is  intended as a mapping of Agnai's common generation setting keys to what each adapter's downstream API calls them (ie: `maxTokens` is the Agnai setting, `max_tokens_to_sample` is presumably what Claude calls it).